src/FakeItEasy/Raise.cs

```diff
@@ -69,5 +69,21 @@ public static TEventHandler With<TEventHandler>(params object[] arguments)
         {
             return new DelegateRaiser<TEventHandler>(arguments, ArgumentProviderMap);
         }

```

Comment:
We could make it go away, but it's unrelated to this change. Since it's breaking, I think we should do it separately.